### PR TITLE
add centraldashboard application CR

### PR DESCRIPTION
**Pull request by kkasravi**
adds a [centraldashboard application CR](https://github.com/kubeflow/manifests/issues/126)

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3557)

### CODE REVIEW

File: bootstrap/config/overlays/application/kfctl_default.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ spec:
     application:
     - name: overlay
       value: application
-    jupyter:
+    centraldashboard:
```
Comment (jlewi) [inline bootstrap/config/overlays/application/kfctl_default.yaml]: Was "jupyter" a bug?

Comment (kkasravi) [reply bootstrap/config/overlays/application/kfctl_default.yaml]: We’re removing this component so yes

**Pull request merged**